docs/index.rst

```diff
@@ -0,0 +1,14 @@
+.. NRI Plugins documentation master file
+
+Welcome to NRI Plugins documentation
+====================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   contributing.md
+
+   resource-policy/index.rst
```

Comment:
I'm getting an error on this line:

```
nri-plugins$ make html
...
...nri-plugins/docs/index.rst:6:toctree contains reference to nonexisting document 'docs/resource-policy /index'
```

Should there be `docs/resource-policy/index.rst` or should this refer to some other file in `resource-policy`?

Reply:
Indeed, there should and I have it which means that I just forgot to push it to git tree. I will submit updated PR.